documentation/source/library_reference.rst

```diff
@@ -139,6 +139,7 @@ Clock
 -----
 
 .. autoclass:: cocotb.clock.Clock
+    :noindex:
```

Comment:
Where's the other reference? A comment might be good here

Reply:
Done.